<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyticsWindow</class>
 <widget class="QDialog" name="AnalyticsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>722</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gameplay Analytics</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../tft_icon.png</normaloff>../../tft_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 94);
color: rgb(255, 170, 0);

font: MS Sans Serif;
font-size: 20px;</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane{
	border: 0;
}

QTabBar::tab {
	font-size: 16px;
	font-weight: bold;
	height: 30px;
	width: 120px;
   background: rgb(0, 0, 94);
	border: 2px solid;
	border-color: rgb(255, 170, 0);
	color: rgb(255, 170, 0);
	padding: 10px;
	margin: 10px;
	margin-right: 20px;
	border-radius: 27px;
}

QTabBar::tab:hover {
   background: rgb(255, 170, 0);
	color: rgb(0, 0, 94);
}

QTabBar::tab:selected {
	color: rgb(0, 0, 94);
    background: rgb(255, 170, 0);
}
</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_overview">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Overview</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_nameAndRegion">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QLabel" name="label_data_region">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>-1</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font-style: italic;
font-size: 40px;</string>
            </property>
            <property name="text">
             <string>placeholder</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_data_summonerName">
            <property name="styleSheet">
             <string notr="true">font-size: 80px;</string>
            </property>
            <property name="text">
             <string>placeholder</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QLabel" name="label_picture_profileIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid;
border-color: rgb(255, 170, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="frame_summary">
         <property name="styleSheet">
          <string notr="true">font-size: 30px;
color: white;
text-shadow:
		-1px -1px 0 #000,
		1px -1px 0 #000,
		-1px 1px 0 #000,
		1px 1px 0 #000;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QLabel" name="label_data_tier">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>With x LP</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_data_leaguePoints">
            <property name="text">
             <string>Averages x Place</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QLabel" name="label_picture_ranked">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid;
border-color: rgb(255, 170, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_recentGames">
            <property name="text">
             <string>Ranked Tier is </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_winRate_2">
            <property name="text">
             <string>Win Rate of x%</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_visualLine">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid;
border-color: rgb(255, 170, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_statistics">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	border: 6px solid;
	padding: 10px;
	margin: 2px;
	border-color: rgb(255, 170, 0);;
}
QWidget {
	margin: 10px;
}</string>
      </property>
      <attribute name="title">
       <string>Statistics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <widget class="QGroupBox" name="box_summonerData">
         <property name="title">
          <string>Summoner</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_summonerLevel">
            <property name="text">
             <string>Summoner Level: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_data_summonerLevel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_activity">
            <property name="text">
             <string>Active: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_data_activity">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="box_teamCompData">
         <property name="title">
          <string>Team Compositions</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_favorite">
            <property name="text">
             <string>Favorite:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_data_favorite">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_favorite2">
            <property name="text">
             <string>2nd Favorite: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_data_favorite2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="box_rankedData">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Ranked (Season 2)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QLabel" name="label_data_gamesPlayed">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_gamesPlayed">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Games Played: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_data_wins">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="labeL_data_placementHistory">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_placementHistory">
            <property name="text">
             <string>Placement History:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_winRate">
            <property name="text">
             <string>Win Rate:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_data_winRate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labeL_losses">
            <property name="text">
             <string>Losses:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_data_hotStreak">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_wins">
            <property name="text">
             <string>Wins:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_hotStreak">
            <property name="text">
             <string>Hot Streak:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_data_losses">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_MatchHistory">
      <attribute name="title">
       <string>LP History</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_LPHistory">
      <attribute name="title">
       <string>Match History</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Team Comps</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Top 10</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
